vertical-pod-autoscaler/pkg/target/controller_fetcher/controller_fetcher.go

```diff
@@ -290,7 +290,7 @@ func (f *controllerFetcher) getOwnerForScaleResource(ctx context.Context, groupK
 		// Some pods specify nodes as their owners. This causes performance problems
 		// in big clusters when VPA tries to get all nodes. We know nodes aren't
 		// valid controllers so we can skip trying to fetch them.
-		return nil, fmt.Errorf("node is not a valid owner")
+		return nil, nil
```

Comment:
Do we really want to mute this error?  
I saw your comment [here](https://github.com/kubernetes/autoscaler/issues/7378#issuecomment-2483161609) where you mentioned that it shouldn’t bubble up as an error. (so just wondering why )

Reply:
I'm a little bit torn about the correct solution here, but this isn't an error. It's a choice by the VPA to not support pods that are owned by nodes.
The pod being created is also not one that the VPA cares about, so it ends up being a lot of noise in the logs.

Reply:
ok I agree. If it’s not documented, we should make sure to document it